uportal-war/src/main/data/required_entities/entity-type/org.apereo.portal.security.IPermissionSet.entity-type.xml

```diff
@@ -20,6 +20,6 @@
 
 -->
 <entity-type script="classpath://org/jasig/portal/io/import-entity-type_v3-2.crn">
-  <name>org.apereo.portal.security.IPermissionSet</name>
+  <name>org.jasig.portal.security.IPermissionSet</name>
```

Comment:
Yep -- that was a mistake that crept in when I was trying to erase _unwanted_ changes the IDE had made auto-magically to `uportal-war/src/main/data`.  Should be fixed already.